### PR TITLE
Altitude plugin

### PR DESCRIPTION
**Pull request by AndreasAntener**
@vooon Added a new plugin for the altitude message. PX4 implementation is in the queue. https://github.com/PX4/Firmware/pull/3234

### CODE REVIEW

File: mavros/src/plugins/altitude.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/**
+ * @brief Altitude plugin
+ * @file altitude.cpp
+ * @author Andreas Antener <andreas@uaventure.com>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2015 Vladimir Ermakov.
```
Comment (vooon) [inline mavros/src/plugins/altitude.cpp]: Why me?

Comment (AndreasAntener) [inline mavros/src/plugins/altitude.cpp]: Thanks, copy paste and such..

File: mavros_msgs/msg/Altitude.msg [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+#
+# https://pixhawk.ethz.ch/mavlink/#ALTITUDE
+
+std_msgs/Header header
+
+float32 altitude_monotonic
+float32 altitude_amsl
+float32 altitude_local
+float32 altitude_relative
+float32 altitude_terrain
```
Comment (vooon) [inline mavros_msgs/msg/Altitude.msg]: Why to repeat `altitude`?

Altitude.monotonic || Altitude.terrain?

Comment (vooon) [inline mavros_msgs/msg/Altitude.msg]: @LorenzMeier how old this message? Why it lack timestamp?

Comment (AndreasAntener) [inline mavros_msgs/msg/Altitude.msg]: You're right, makes no sense

**Comment by AndreasAntener** [comment]
@vooon @LorenzMeier About the missing timestamp: is it just to factor out transmission delays? Or about syncing sensor data? (the later would require multiple timestamps then since the data comes from multiple sources)

**Comment by LorenzMeier** [comment]
Transmission. Let's add it

**Comment by AndreasAntener** [comment]
@vooon This is good to go with latest mavlink

**Comment by vooon** [comment]
Ok, i will release mavlink update.

- Indigo: https://github.com/ros/rosdistro/pull/9867
- Jade:

**Pull request merged**